src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { FilterType } from '../../types/FilterType';
+import { useTodos } from '../../utils/TodoContext';
+import { useClearCompleted } from '../../hooks/useClearCompleted';
+import classNames from 'classnames';
+
+export const Footer: React.FC = () => {
+  const { todos, filter, setFilter } = useTodos();
+  const { handleClearCompleted } = useClearCompleted();
+
+  if (!todos?.length) {
+    return null;
+  }
+
+  const activeTodosCount = todos.filter(todo => !todo.completed).length;
+  const completedTodosCount = todos.filter(todo => todo.completed).length;
```

Comment:
I guess you don't have to filter out completedTodos. You can just now do some maths with activeTodosCount and all todos so 
`todos.length - activeTodosCount = completedTodosCount`